client/app/domain/dns/lock/domain-dns-lock.controller.js

```diff
@@ -19,7 +19,10 @@ angular.module("controllers").controller(
             return this.Domain
                 .updateNameServerType(this.$stateParams.productId, "hosted")
                 .then(() => this.Alerter.success(this.$scope.i18n.domain_tab_DNS_lock_success, this.$scope.alerts.main))
-                .catch((err) => this.Alerter.alertFromSWS(this.$scope.i18n.domain_tab_DNS_lock_error, err, this.$scope.alerts.main))
+                .catch((err) => {
+                    _.set(err, "type", err.type || "ERROR");
+                    this.Alerter.alertFromSWS(this.$scope.i18n.domain_tab_DNS_lock_error, err, this.$scope.alerts.main);
```

Comment:
We should have different methods for error messages, information messages, etc...